lib/beaker/dsl/install_utils.rb

```diff
@@ -901,14 +901,17 @@ def install_puppet_from_msi( host, opts )
           dest = "C:\\Windows\\Temp\\#{host['dist']}.msi"
 
           on host, "set PATH=\"%PATH%;#{host['puppetbindir']}\""
-          on host, "setx PATH \"%PATH%;#{host['puppetbindir']}\""
```

Comment:
You should be able to remove both set and setx as the pathing is done during host provisioning steps.